scripts/cc-cli/src/commands/newSeed.ts

```diff
@@ -0,0 +1,27 @@
+import { Command, OptionValues } from "commander";
+import { mnemonicGenerate } from "@polkadot/util-crypto";
+import { writeFileSync } from "fs";
+
+export function makeNewSeedCommand() {
+  const cmd = new Command("new");
+  cmd.description("Create new mnemonic seed");
+  cmd.option("-l, --length [word-length]", "Specify the amount of words");
+  cmd.option("-s, --save [file-name]", "Save the new stash seed to a file");
+  cmd.action(newSeedAction);
+  return cmd;
+}
+
+function newSeedAction(options: OptionValues) {
+  console.log("Creating new mnemonic...");
+  const mnemonic = options.lenght
+    ? mnemonicGenerate(options.lenght)
```

Comment:
```suggestion
  const mnemonic = options.length
    ? mnemonicGenerate(options.length)
```
Gotta love dynamically created properties🥲